firefox-ios/firefox-ios-tests/Tests/ClientTests/Coordinators/RouteTests.swift

```diff
@@ -7,6 +7,16 @@ import XCTest
 
 @MainActor
 class RouteTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        DependencyHelperMock().bootstrapDependencies()
+    }
+
+    override func tearDown() {
+        DependencyHelperMock().reset()
+        super.tearDown()
+    }
```

Comment:
This is so we avoid changes when https://github.com/mozilla-mobile/firefox-ios/pull/31261 will be merged 👀 
```suggestion
    override func setUp() async throws {
        try await super.setUp()
        DependencyHelperMock().bootstrapDependencies()
    }

    override func tearDown() async throws {
        DependencyHelperMock().reset()
        try await super.tearDown()
    }
```